lib/intarray.ml

```diff
@@ -0,0 +1,68 @@
+module Int64Array : sig
+  module Array : sig
```

Comment:
Why do we need this module inside a module?

Reply:
The syntax `a.(42)` is sugar for `Array.get a 42`, using whatever `Array` module is in scope. If you define a module with a nested module called `Array` like this, then you can `open Int64Array` to use the array syntax. (Yes, this is horrible)

Reply:
Ah. I see. That works :-)